Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
+Hi there! This guide aims to teach people unfamiliar with Github how to suggest edits to the [Tanzu Agile Practices site](https://tanzu.vmware.com/developer/practices/). Follow the steps below to get started!
+
+
+# Workflow overview
+
+It takes many clicks to suggest an edit, but these are the 5 big stages you have to go through.
+
+1. **Prepare**: Type up your suggested edits 
+1. **Fork**: Forking just means creating a copy of the practices site code for your GitHub account
+1. **Edit**: Copy and paste suggested edits into the code in your copy of the practices site through the GitHub UI and save it in a branch
+1. **Submit a Pull Request**: Submitting a Pull Request is the technical term for submitting the suggested edits to be reviewed, approved by someone else and updated on the site. It is so named because the reviewer will be 'pulling' your code into the vmware repository to merge it. 
```

Comment:
```suggestion
1. **Submit a Pull Request**: Submitting a Pull Request is the technical term for submitting the suggested edits to be reviewed, approved by someone else and updated on the site. It is so named because the reviewer will be 'pulling' your code into the VMware repository to merge it. 
```